tests/LoginAPI.spec.js

```diff
@@ -0,0 +1,118 @@
+import { test, expect } from "../PageObjects/BaseObject.js";
+import credentials from "../Fixtures/credentials.json";
+import dotenv from "dotenv";
+import path from "path";
+dotenv.config({ path: path.resolve(__dirname, "../.env") });
+
+let baseUrl;
+
+test.describe("Positive API test cases for Login", async () => {
+  test.beforeEach(async () => {
+    baseUrl = process.env.BASEURLAPI;
+  });
+
+  test("Login with valid credentials", async ({ loginAPI }) => {
```

Comment:
when you create one function to do all for you, test will look like this: 

await authAPI.login({}) -> that will be success login test

other:
await authAPI.login({
userName : "",
statusCode: 422
})

check error message